code/public/index.html

```diff
@@ -13,7 +13,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title>Top Rated Movies</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="code/src/assets/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="code/src/assets/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="code/src/assets/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
```

Comment:
I cannot see your favicon on the published Netlify link. I think that placing it in the public folder instead will make it show https://medium.com/@jenniferdobak/the-public-folder-and-favicons-in-create-react-app-8dc2cc1d492b